### PR TITLE
change(doc): Fix pool name, links, make commands and formatting consistent

### PR DESCRIPTION
**Pull request by teor2345**
This PR:
* renames `pool_configs/zclassic.json` to `zcash_testnet.json`
* links to the correct Zebra build instructions
* formats commands the same way
* formats whitespace the same way

### CODE REVIEW

File: zebra-instructions.md [hunk 1]
```diff
@@ -124,16 +120,15 @@ Install `s-nomp`:
     nvm use 8.17.0
     ```
 
-5. Update dependencies and install:
+4. Update dependencies and install:
```
Comment (teor2345) [inline zebra-instructions.md]:
```suggestion
5. Update dependencies and install:
```

File: zebra-instructions.md [hunk 2]
```diff
@@ -103,15 +99,15 @@ Install dependencies:
...
     sudo apt install libsodium-dev
     ```
 
 Install `s-nomp`:
-1. `git clone https://github.com/ZcashFoundation/s-nomp && cd s-nomp`
-3. Use the Zebra configs: `git checkout zebra-mining`
+1. `git clone https://github.com/ZcashFoundation/s-nomp`
+2. `cd s-nomp`
+3. Use the Zebra fixes: `git checkout zebra-mining`
     * TODO: change the fork in `package.json` to ZcashFoundation
```
Comment (teor2345) [inline zebra-instructions.md]: This is the only remaining TODO, I can push it in a separate code PR

Comment (teor2345) [reply zebra-instructions.md]: This is now PR #7.

```suggestion
```

**Review by arya2 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**